### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by lainslie**
OASIS brain DCGAN implemented in Tensorflow. Dependencies and usage have been detailed in the readme.

Images within README have been uploaded online, and are not utilising the available folder. If in time when viewing the readme the photos are no longer available for some unknown reason, please see the submitted pdf for results images.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Work submitted before deadline
- Code appear valid for Hard Difficulty 
- Results are good
- Commit messages can be a bit more specific 

Other Feedback
- You should put your readme.md (instead of the PDF version of it) in your project directory.

**Comment by rexsw** [comment]
Code looks good for hard difficulty

For next time;

don't have lines like cross_entropy = tf.keras.losses.BinaryCrossentropy(from_logits=False) just floating around put them in a funcation 

Tensorflow has functions for image opening and more so there’s no need to also use PIL and numpy in your project.

File: recognition/problem6/load_data.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import tensorflow as tf
```
Comment (shakes76) [inline recognition/problem6/load_data.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
Labels generated instead of MR images
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. You should have you're own readme and not overwrite recognition folder one. Please fix**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
